consensus/core/src/synchronizer.rs

```diff
@@ -41,10 +41,15 @@ use crate::{
 /// The number of concurrent fetch blocks requests per authority
 const FETCH_BLOCKS_CONCURRENCY: usize = 5;
 
+/// Timeouts when fetching blocks.
 const FETCH_REQUEST_TIMEOUT: Duration = Duration::from_millis(2_000);
-
 const FETCH_FROM_PEERS_TIMEOUT: Duration = Duration::from_millis(4_000);
 
+/// Max number of blocks to fetch per request.
+/// This value should be chosen so even with blocks at max size, the requests
+/// can finish on hosts with good network using the timeouts above.
+const MAX_BLOCKS_PER_FETCH: usize = 32;
```

Comment:
Saw on the other PR that this was set to `20`. `32` looks better.

Reply:
Yeah especially there shouldn't be many more missing blocks compared to # of authorities in the committee.